scipy/interpolate/rbf.py

```diff
@@ -218,8 +249,11 @@ def __init__(self, *args, **kwargs):
         for item, value in kwargs.items():
             setattr(self, item, value)
 
+        r = self._call_norm(self.xi, self.xi)
         self.A = self._init_function(r) - np.eye(self.N)*self.smooth
-        self.nodes = linalg.solve(self.A, self.di)
+        self.A_aug = self._generate_augmented_matrix(self.xi.T)
```

Comment:
This should not be stored on the instance: the matrix is only used once and can be large. If really needs be, it can be made into a property, or just discarded.